lib/chef/knife/ec2_server_create.rb

```diff
@@ -62,14 +60,14 @@ class Ec2ServerCreate < Chef::Knife::Bootstrap
         description: "The security groups for this server; not allowed when using VPC",
         proc: Proc.new { |groups| groups.split(",") }
 
-      option :security_group_id,
+      option :security_group_ids,
```

Comment:
Do we even need to change this if it just accumulates it up?

Reply:
It has to be this way to be backcompatible with the prior config setting and with the API options, so yes, this code is correct.  The name change here matches the old code in the accumulator.